docs/deployment/setup-seed.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Setting up a Seed Cluster
+description: "How to configure a Kubernetes cluster as a Gardener seed"
```

Comment:
I think we don't need this one because we already have
- https://github.com/gardener/gardener/blob/master/docs/usage/seed_settings.md
- https://github.com/gardener/gardener/blob/master/docs/usage/managed_seed.md
- https://github.com/gardener/gardener/blob/master/docs/usage/shooted_seed.md